core/src/test/java/org/apache/iceberg/TestMetadataTableScans.java

```diff
@@ -527,6 +527,63 @@ public void testDeleteFilesTableSelection() throws IOException {
     Assert.assertEquals(expected, scan.schema().asStruct());
   }
 
+  @Test
+  public void testFilesTableReadableMetricsSchema() {
+
+    Table filesTable = new FilesTable(table.ops(), table);
+    Types.StructType actual = filesTable.newScan().schema().select("readable_metrics").asStruct();
+
```

Comment:
Just to make this a little easier in the future you may just want to do something like

firstAssigned = (schema.highestId - 15)
Then do 
1001 = firstAssigned +1; ....

not sure this really helps that much though

Reply:
Done